hs_core/views/resource_rest_api.py

```diff
@@ -916,6 +1059,7 @@ def dispatch(self, *args, **kwargs):
 
         # get the hydroshare resource id from the metadata
         hs_res_id = metadata.get('hs_res_id')
+        username_from_client = metadata.get('username')
```

Comment:
Should this be in the except block before the assertion?

Reply:
removed it